Live/LiveEventWithDVR/README.md

```diff
@@ -1,29 +1,31 @@
-# Live event with DVR
+# Live Event with DVR
 
-This sample demonstrates how to create and use LiveEvents and LiveOutputs in the v3 Media Services API. It performs the following tasks:
+This sample demonstrates how to create and use Live Events and Live Outputs in the v3 Media Services API. It performs the following tasks:
 
 1. Creates a "standard" pass-through LiveEvent.
 1. Starts monitoring the LiveEvent using Event Grid and Event Hub.
 1. Creates an Asset and an AssetFilter with 5 minutes sliding window and 30 seconds seek back time.
-1. Creates a LiveOutput with 25 hours time span.
-1. Creates a StreamingLocator and associate the AssetFilter with it.
-1. Prints urls for the LiveEvent.
-1. Prints playback urls for the event archive after the LiveEvent stops.
+1. Creates a Live Output with 25 hours time span.
+1. Creates a Streaming Locator and associate the Asset Filter with it.
+1. Prints URLs for the Live Event.
+1. Prints playback URLs for the event archive after the Live Event stops.
 
 ## Live events and live outputs in Media Services
 
-This sample shows how to create a "standard" pass-through live event.  There are several types of live events available at different pricing points.  This is the cheapest option, and takes the source video without transcoding it any further and just packages it for HLS and DASH delivery.  You can also choose "basic" pass-through, or from the two cloud transcoding live events for 720P and 1080P adaptive bitrate output.  
+This sample shows how to create a "standard" pass-through live event. There are several types of live events available at different pricing points.  This is the cheapest option, and takes the source video without transcoding it any further and just packages it for HLS and DASH delivery.  You can also choose "basic" pass-through, or from the two cloud transcoding live events for 720P and 1080P adaptive bitrate output.
 For details on the various types of live events see the article [Live events and live outputs in Media Services.](https://learn.microsoft.com/azure/media-services/latest/live-event-outputs-concept)
 
 ## Prerequisites
 
 Required Assemblies:
 
 * Azure.Identity
-* Azure.Messaging.EventGrid
-* Azure.Messaging.EventHubs.Processor
+* Azure.ResourceManager.EventGrid
```

Comment:
I thought you were removing Event Grid support?

Reply:
I was going to move it to a separate sample, but I realized I could instead make the Event Grid integration optional and hide all the details in a separate class.